doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+      * :c:func:`nrf_cloud_pending_fota_job_process` function that processes the state of pending FOTA jobs.
+      * :c:func:`nrf_cloud_handle_error_message` function that handles error message responses (MQTT) from nRF Cloud.
+
+    * Updated:
+
+      * Shadow data behavior during the connection process.
+        The data is now sent to the application even if no ``"config"`` section is present.
+      * The application can now send shadow updates earlier in the connection process.
+      * nRF Cloud error message responses to location service MQTT requests are now handled.
+      * The value of the :kconfig:option:`NRF_CLOUD_HOST_NAME` option is now "mqtt.nrfcloud.com".
+      * Removed support for the Cloud API.
+
+    * Fixed the validation of bootloader FOTA updates.
+
+  * :ref:`lib_aws_iot` library:
+
+    * Updated:
+
+      * Renamed ``aws_iot_topic_type`` to ``aws_iot_shadow_topic_type`` and removed ``AWS_IOT_SHADOW_TOPIC_UNKNOWN``.
+      * Removed support for the Cloud API.
+
+  * :ref:`lib_lwm2m_client_utils` library:
+
+    * Added support for using location assistance when using the Coiote LwM2M server.
+    * Updated the library to store credentials and server settings permanently on bootstrap.
+    * Updated the library to let an application control the network connection state.
+
+  * :ref:`lib_azure_iot_hub` library:
+
+    * Added :kconfig:option:`CONFIG_AZURE_IOT_HUB_NATIVE_TLS` option to configure the socket to be native for TLS instead of offloading TLS operations to the modem.
+    * Removed support for the Cloud API.
+
+ * :ref:`lib_nrf_cloud_pgps` library:
+
+    * Added:
+
+      * The passing of the current prediction, if one is available, along with the ``PGPS_EVT_READY`` event.
+      * The following three ways to define the storage location in the flash memory:
+
+        * A dedicated P-GPS partition, enabled with the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_PARTITION` option.
+        * The use of the MCUboot secondary partition as storage, enabled with the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_MCUBOOT_SECONDARY` option.
+        * An application-specific storage, enabled with the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_CUSTOM` option.
+
+ * :ref:`lib_nrf_cloud_agps` library:
+
+    * Fixed premature assistance suppression when the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS_FILTERED` option is enabled.
+      Added a 10 minute margin of error to ensure A-GPS assistance is downloaded every two hours even if the modem requests assistance a little early.
+
+* Removed:
+
+  * Cloud API library.
+
+Libraries for NFC
+-----------------
+
+* Updated:
+
+  * :ref:`nfc_ndef_parser_readme`:
+
+    * Updated:
+
+      * :c:func:`nfc_ndef_msg_parse` with a fix to the declaration, a new assertion to avoid a potential usage fault, and added a note in the function documentation.
```

Comment:
```suggestion
      * :c:func:`nfc_ndef_msg_parse` with a fix to the declaration, a new assertion to avoid a potential usage fault, and added a note in the API documentation.
```

Reply:
done